core-sdk-samples/mp-sideloaded-kit-example/package.json

```diff
@@ -0,0 +1,40 @@
+{
```

Comment:
I thought you were going to move the kit into a sibling folder of `core-sdk-samples` rather than a child directory. Was I mistaken?

Reply:
accidental